components/Spaces/CallTabs/Music/Track.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+/* eslint-disable @next/next/no-img-element */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
why we no want this?

Reply:
will add later. Too much work to do in this pr ahaha